alembic/versions/52decb727dcb_remove_pending_companies.py

```diff
@@ -0,0 +1,13 @@
+revision = '52decb727dcb'
+down_revision = 'beba44b3efb'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.drop_table('old_company')
+    op.drop_table('old_company_link')
+
+def downgrade():
+    pass
```

Comment:
The revision should recreate the deleted tables on `downgrade`. Did you create it manually? You have the option to autogenerate it:

    ./manage.py db alembic revision -- --autogenerate -m 'commit message'